src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -107,11 +107,8 @@ afterAll(() => {
 });
 
 describe('Create Image Wizard', () => {
-    beforeEach(async () => {
```

Comment:
https://jestjs.io/docs/api#beforeeachfn-timeout beforeEach should be fine to use as a setup no? `beforeAll` isn't used.

Reply:
So, according to [the docs](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/no-render-in-setup.md), it is best to not render in the beforeEach. Figuring out why this is a rule is a little tricky and I would also be fine setting a rule to ignore this case. From what I've read we shouldn't separate the render from our tests. The render call should be made in each test as it is when calling `setUp` now. This supposedly helps with cleanup and, overall, this rule is meant to also encourage writing fewer but longer tests.